packages/labs/motion/src/demo/styles.css.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import {css} from 'lit';
+
+export const styles = css`
+  :host {
+    font-family: Arial, 'sans serif';
+    display: block;
+  }
+  .hooks-main {
+    width: 100%;
+    height: 100%;
+    background: lightblue;
+    overflow: hidden;
+    display: flex;
+    align-items: center;
+    justify-content: center;
```

Comment:
This can be deleted because the elements within this container end up being absolutely positioned.